crypto/hacl_hash.h

```diff
@@ -0,0 +1,392 @@
+	(uint32_t)0x78a5636fU, (uint32_t)0x84c87814U, (uint32_t)0x8cc70208U,
+	(uint32_t)0x90befffaU, (uint32_t)0xa4506cebU, (uint32_t)0xbef9a3f7U,
+	(uint32_t)0xc67178f2U
+};
+
+static const uint64_t Hacl_Impl_SHA2_Generic_k384_512[80U] = {
+	(uint64_t)0x428a2f98d728ae22U, (uint64_t)0x7137449123ef65cdU,
+	(uint64_t)0xb5c0fbcfec4d3b2fU, (uint64_t)0xe9b5dba58189dbbcU,
+	(uint64_t)0x3956c25bf348b538U, (uint64_t)0x59f111f1b605d019U,
+	(uint64_t)0x923f82a4af194f9bU, (uint64_t)0xab1c5ed5da6d8118U,
+	(uint64_t)0xd807aa98a3030242U, (uint64_t)0x12835b0145706fbeU,
+	(uint64_t)0x243185be4ee4b28cU, (uint64_t)0x550c7dc3d5ffb4e2U,
+	(uint64_t)0x72be5d74f27b896fU, (uint64_t)0x80deb1fe3b1696b1U,
+	(uint64_t)0x9bdc06a725c71235U, (uint64_t)0xc19bf174cf692694U,
+	(uint64_t)0xe49b69c19ef14ad2U, (uint64_t)0xefbe4786384f25e3U,
+	(uint64_t)0x0fc19dc68b8cd5b5U, (uint64_t)0x240ca1cc77ac9c65U,
+	(uint64_t)0x2de92c6f592b0275U, (uint64_t)0x4a7484aa6ea6e483U,
+	(uint64_t)0x5cb0a9dcbd41fbd4U, (uint64_t)0x76f988da831153b5U,
+	(uint64_t)0x983e5152ee66dfabU, (uint64_t)0xa831c66d2db43210U,
+	(uint64_t)0xb00327c898fb213fU, (uint64_t)0xbf597fc7beef0ee4U,
+	(uint64_t)0xc6e00bf33da88fc2U, (uint64_t)0xd5a79147930aa725U,
+	(uint64_t)0x06ca6351e003826fU, (uint64_t)0x142929670a0e6e70U,
+	(uint64_t)0x27b70a8546d22ffcU, (uint64_t)0x2e1b21385c26c926U,
+	(uint64_t)0x4d2c6dfc5ac42aedU, (uint64_t)0x53380d139d95b3dfU,
+	(uint64_t)0x650a73548baf63deU, (uint64_t)0x766a0abb3c77b2a8U,
+	(uint64_t)0x81c2c92e47edaee6U, (uint64_t)0x92722c851482353bU,
+	(uint64_t)0xa2bfe8a14cf10364U, (uint64_t)0xa81a664bbc423001U,
+	(uint64_t)0xc24b8b70d0f89791U, (uint64_t)0xc76c51a30654be30U,
+	(uint64_t)0xd192e819d6ef5218U, (uint64_t)0xd69906245565a910U,
+	(uint64_t)0xf40e35855771202aU, (uint64_t)0x106aa07032bbd1b8U,
+	(uint64_t)0x19a4c116b8d2d0c8U, (uint64_t)0x1e376c085141ab53U,
+	(uint64_t)0x2748774cdf8eeb99U, (uint64_t)0x34b0bcb5e19b48a8U,
+	(uint64_t)0x391c0cb3c5c95a63U, (uint64_t)0x4ed8aa4ae3418acbU,
+	(uint64_t)0x5b9cca4f7763e373U, (uint64_t)0x682e6ff3d6b2b8a3U,
+	(uint64_t)0x748f82ee5defb2fcU, (uint64_t)0x78a5636f43172f60U,
+	(uint64_t)0x84c87814a1f0ab72U, (uint64_t)0x8cc702081a6439ecU,
+	(uint64_t)0x90befffa23631e28U, (uint64_t)0xa4506cebde82bde9U,
+	(uint64_t)0xbef9a3f7b2c67915U, (uint64_t)0xc67178f2e372532bU,
+	(uint64_t)0xca273eceea26619cU, (uint64_t)0xd186b8c721c0c207U,
+	(uint64_t)0xeada7dd6cde0eb1eU, (uint64_t)0xf57d4f7fee6ed178U,
+	(uint64_t)0x06f067aa72176fbaU, (uint64_t)0x0a637dc5a2c898a6U,
+	(uint64_t)0x113f9804bef90daeU, (uint64_t)0x1b710b35131c471bU,
+	(uint64_t)0x28db77f523047d84U, (uint64_t)0x32caab7b40c72493U,
+	(uint64_t)0x3c9ebe0a15c9bebcU, (uint64_t)0x431d67c49c100d4cU,
+	(uint64_t)0x4cc5d4becb3e42b6U, (uint64_t)0x597f299cfc657e2aU,
+	(uint64_t)0x5fcb6fab3ad6faecU, (uint64_t)0x6c44198c4a475817U
+};
+
+void Hacl_SHA2_Scalar32_sha256_init(uint32_t *hash);
+
+void Hacl_SHA2_Scalar32_sha256_update_nblocks(uint32_t len, uint8_t *b,
+					      uint32_t *st);
```

Comment:
Align to the open parenthesis. -- This is solved by clang-format in cf-zeta branch.

```suggestion
void Hacl_SHA2_Scalar32_sha256_update_nblocks(uint32_t len, uint8_t *b,
					                          uint32_t *st);
```